tensorflow/python/keras/callbacks.py

```diff
@@ -1220,20 +1225,21 @@ def on_epoch_end(self, epoch, logs=None):
     current = self.get_monitor_value(logs)
     if current is None:
       return
-    if self.monitor_op(current - self.min_delta, self.best):
-      self.best = current
-      self.wait = 0
-      if self.restore_best_weights:
-        self.best_weights = self.model.get_weights()
-    else:
-      self.wait += 1
-      if self.wait >= self.patience:
-        self.stopped_epoch = epoch
-        self.model.stop_training = True
+    if epoch > self.min_epochs:
```

Comment:
should this be `epoch == self.min_epochs`?

Reply:
Thanks for the comment!

I think it should be `epoch > self.min_epochs` because the goal is to check if the current epoch is after the minimum number of epochs specified. If the check passes, we can execute the rest of the `EarlyStopping` code. This way, the code is always executed for every epoch after `self.min_epochs`